src/app/policies/components/PolicyConnections.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <h2>Dataplanes</h2>
+  <h3>{{ t('policies.detail.affected_dpps') }}</h3>
```

Comment:
I think these things should be move into the `*View.vue`. We should try to never have `h*`s in components as it makes them not reusable. I'm pretty sure we could get rid of the `PolicyDetails` component entirely?

Reply:
I moved the heading into the PolicyDetails component, but I want to keep this split of *View and (mostly) presentational components.